### PR TITLE
give unique Core icons to Equipment items

### PR DESCRIPTION
**Pull request by ammalagonc**
- v2024-08-18

### CODE REVIEW

File: packs/equipment/camouflaging-chromatophores-greater.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "v2aOpKhNP0uIZFER",
-    "img": "icons/creatures/reptiles/chameleon-camouflage-green-brown.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/camouflaging-chromatophores-greater.json:3]: Was this intentional?

File: packs/equipment/camouflaging-chromatophores-lesser.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "RmIA4czgAOH8Ml7e",
-    "img": "icons/creatures/reptiles/chameleon-camouflage-green-brown.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/camouflaging-chromatophores-lesser.json:3]: And here?

File: packs/equipment/doctrine-of-blissful-eternity.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "OVyGHaHjYI2DBV0h",
-    "img": "icons/sundries/books/book-leaves-circle.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/doctrine-of-blissful-eternity.json:3]: And here?

File: packs/equipment/eidolon-cape.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "nxAMYJrG96yn9iBq",
-    "img": "icons/equipment/back/cape-layered-purple.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/eidolon-cape.json:3]: And here?

File: packs/equipment/gills.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "RiChMr22ar4ThfGF",
-    "img": "icons/creatures/fish/fish-carp-green.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/gills.json:3]: And here?

File: packs/equipment/grimoire-of-unknown-necessities.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "oqZszBqqanONR0ng",
-    "img": "icons/sundries/books/book-rounded-red.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/grimoire-of-unknown-necessities.json:3]: And here?

File: packs/equipment/singing-stone.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "MM44urhBdG0x98bc",
-    "img": "icons/commodities/stone/geode-raw-beige.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/singing-stone.json:3]: And here?

File: packs/equipment/violet-ray.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "pXOPMJC9NAuwXz3w",
-    "img": "icons/commodities/tech/antenna-powered-purple.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```
Comment (simonward) [inline packs/equipment/violet-ray.json:3]: And here?

File: packs/equipment/camouflaging-chromatophores-greater.json (hunk 1 above)
Comment (ammalagonc) [reply packs/equipment/camouflaging-chromatophores-greater.json:3]: No, this was not - this undid the previous round - I'll redo the request in a bit.

**Pull request merged**